tests/cover/test_numerics.py

```diff
@@ -122,3 +122,18 @@ def test_issue_725_regression(x):
 @given(decimals(min_value='0.1', max_value='0.3'))
 def test_issue_739_regression(x):
     pass
+
+
+@given(decimals(min_value='10E99998', max_value='10E99999'))
+def test_issue_838_regression(x):
+    # If this hands forever, that's a regression
```

Comment:
Also it would be nice if we could make this fail rather than hang. Maybe we should use https://pypi.python.org/pypi/pytest-timeout ? I'm not sure how it interacts with xdist.

Reply:
Also also, we should include a test that checks for regression on the variant actually reported by a user! (i.e. with `Decimal(0).next_minus()`)

Reply:
Yep, incompatible with Python 3.4 😢, and yep!